client/src/main/scala/drt/client/components/Layout.scala

```diff
@@ -83,7 +86,10 @@ object Layout {
               } else EmptyVdom,
               <.div(^.className := "topbar",
                 <.div(^.className := "main-logo"),
-                <.div(^.className := "alerts", AlertsComponent())
+                AlertsComponent(),
+                <.div(^.style := js.Dictionary("display" -> "flex", "alignItems" -> "center", "width" -> "320px", "justifyContent" -> "flex-end"),
```

Comment:
We don't need the `justifyContent` or the `width` here. I reckon we can probably also remove the `span` as all it's doing is the padding which can be applied to the div instead.